tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/ImdsV2Tests.cs

```diff
@@ -175,9 +175,11 @@ public async Task BearerTokenTokenIsPerIdentity(
                 #endregion Identity 1
 
                 #region Identity 2
-                var managedIdentityApp2 = await CreateManagedIdentityAsync(httpManager, userAssignedIdentityId, userAssignedId, addProbeMock: false, addSourceCheck: false).ConfigureAwait(false); // source is already cached
+                UserAssignedIdentityId identity2Type = UserAssignedIdentityId.ClientId;
+                string identity2Id = TestConstants.ClientId2;
```

Comment:
You'll need to include these in the DataRow so that a resourceId and Object can be used instead of ClientId every time.

Reply:
Done for the type. 

I don't think the ID needs to set to some specific value based on the type. The ID is just a string, so as long as it's different from Identity1, it fits the purpose of this test.

Reply:
Done.